AGPs/AGP-LD1.md

```diff
@@ -0,0 +1,63 @@
+---
+AGP: TBD
+Title: Aragon App Mining
+Author: lkngtn (@lkngtn)
+Status: Draft
+Track: Finance
+Created: 2019-10-02
+---
+
+# AGP-M: Aragon App Mining
+
+## Address of the transfer recipient
+
+TBD: 0xe0d11d5a1982f64ff1d36cd8c2bd315dea127d66 (Agent of pinion.aragonid.eth)
+
+## Amount of the transfer
+
+100K ANT
+50K DAI
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+Quarterly transfers of 100K ANT with no expiration date and beginning after ANV 5 and due the week after each ANV.
+One time transfer of 50K DAI for initial execution and launch of the App Mining program.
+
+## Purpose of the transfer
+
+To fund an "Aragon App Mining" program which rewards developers for creating and maintaining Aragon applications. The policy is inspired by [Blockstack's App Mining program](https://app.co/mining).
+
+A core value proposition of Aragon is that it is a platform which allows users to pull from a library of modular and composable applications to create and operate an organization on-chain. In order to attract application developers to the platform we need to provide a stable opportunity that encourages entrepreneurial developers to understand user needs, develop applications, ship them to mainnet, and to continue to maintain and improve them over time.
+
+At the moment there is no incentive to do this outside of one-off grants. There has been discussion to create monetization models for the eventual release of the App Center, but currently such a system doesn't yet exist, and even if it did the number of users of Aragon is small enough that such a market may not be attractive enough to bring top talent into the space. Users will come if there are high quality apps, but if there are no users, no high quality apps will be built and maintained.
+
+To remedy this I propose we bootstrap the supply side by allocating funds and distributing them to Aragon Application developers based on agreed upon KPIs. We can combine multiple KPIs into an **Aragon App Score**, which we can use to rank applications and then use the ranking to distribute funding to maintainers on a periodic basic.
+
+**The following weighted KPIs would be used to create the Aragon App Score:**
+
+| KPI | Formula | Weight |
+| -------- | -------- | -------- |
+| Utility    | Sum of (# of transactions involving an application ) | 30%  |
+| Value weighted utility    | Sum of (# of transactions involving an application * DAI value of organization's treasury) | 30%    |
+| Popularity    | Sum of (# of installations) | 10%  |
+| Value weighted Popularity    | Sum of (# of installations *  DAI value of organization's treasury) | 30% |
+
+To compute the DAI value of an organizations treasury we would use the *default vault* and compute the DAI value of ETH, ANT, and DAI held by the organization.
```

Comment:
What if there are multiple vaults installed and used for some reason? Which is considered the "default vault"?

Reply:
I didn't see where this question was resolved.

Reply:
The kpi are indicated as for example purposes and will be determined before the next anv. Didn’t feel that it was a relevant enough issue so choose to ignore it for now and resolve it given the timing. As far as I’m aware it is not necessary to address every comment.